src/api/types.ts

```diff
@@ -19,6 +19,7 @@ export interface CliCommands {
   ingest_status: IngestStatusResponse;
   rewrite: RewriteChoice[];
   chat: ChatResponseChoice[];
+  delete: ResponseStatus;
```

Comment:
This is just returning a status and not what you have described in the README.md (status/message).